src/Shared/Infrastructure/RetryStrategy/InfiniteRetryStrategy.php

```
@@ -0,0 +1,32 @@
+ *
+ * Failed messages are retried indefinitely with a fixed delay.
+ * This supports our AP (Availability + Partition tolerance) architecture:
+ * - Events that fail are never lost
+ * - They remain in the failed queue until the issue is resolved
+ * - Metrics + alerts trigger automated fixes via AI agents
+ */
+final readonly class InfiniteRetryStrategy implements RetryStrategyInterface
+{
+    private const RETRY_DELAY_MS = 60000; // 60 seconds
+
+    public function isRetryable(Envelope $message, ?\Throwable $throwable = null): bool
```

PHPMD (2.15.0)

Avoid unused parameters such as '$message'. (undefined)

(UnusedFormalParameter)